tasks/7_random-fraction/random-fraction.js

```diff
@@ -0,0 +1,4 @@
+function randomFraction() {
+  return Math.random();
```

Comment:
Example of `Math.random()` call:
0.8095752234458309
We need to get random of 0.1, 0.2,...,0.9